ert3/evaluator/_evaluator.py

```diff
@@ -185,17 +206,23 @@ def command_location(name: str) -> FilePath:
         }
     ]
 
+    storage_conf = {
+        "type": ensemble.storage_type,
+        "storage_path": evaluation_tmp_dir / ".my_storage",
+    }
+
+    if ensemble.storage_type == "ert_storage":
+        workspace = evaluation_tmp_dir.parent.parent.parent
```

Comment:
This at least requires a comment I think, but ideally we would not depend on a relative placement between the workspace and evaluation folder to find the information we need.